fsspec/implementations/github.py

```diff
@@ -89,6 +97,7 @@ def tags(self):
         """Names of tags in the repo"""
         r = requests.get(
             f"https://api.github.com/repos/{self.org}/{self.repo}/tags",
+            self.timeout,
```

Comment:
Shouldn't this and the ones below all be `timeout=`, rather than a positional argument?

Reply:
Ugh, yes, sorry about that. It should be fixed now.